lib/streamlit/elements/plotly_chart.py

```diff
@@ -542,11 +552,7 @@ def plotly_chart(
         plotly_chart_proto.theme = theme or ""
         plotly_chart_proto.form_id = current_form_id(self.dg)
 
-        config = dict(kwargs.get("config", {}))
-        # Copy over some kwargs to config dict. Plotly does the same in plot().
-        config.setdefault("showLink", kwargs.get("show_link", False))
-        config.setdefault("linkText", kwargs.get("link_text", False))
```

Comment:
Wouldn't this remove the full support for `kwargs`? I think we still want to support the kwargs style in addition to the new config until we actually remove it.

Reply:
Oh no, definitely show a deprecation warning for now but keep `kwargs` support. I think we can already get rid of copying `showLink` and `linkText` to `config`, since those params don't seem to be used.